src/Sylius/Bundle/InstallerBundle/Behat/CliContext.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\InstallerBundle\Behat;
+
+use Sylius\Bundle\InstallerBundle\Command\SetupCommand;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+require_once __DIR__.'../../../../../../vendor/phpunit/phpunit/src/Framework/Assert/Functions.php';
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class CliContext extends DefaultContext
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->application = new Application($this->getKernel());
```

Comment:
Seems like you've encountered an error! :) 

Contexts are instantiated in `ContextFactory`, exactly [at line 80](https://github.com/Behat/Behat/blob/dc6df78862e1c1e1faa7bffd396d7c3179d6749d/src/Behat/Behat/Context/ContextFactory.php#L80). They are initialized just [one line below](https://github.com/Behat/Behat/blob/dc6df78862e1c1e1faa7bffd396d7c3179d6749d/src/Behat/Behat/Context/ContextFactory.php#L81) and `KernelInterface` instance is set by Symfony2Extension context initializer.

To solve this you can register `@BeforeScenario` hook (which has some downsides, especially that it would be called before every scenario in every suite that has `CliContext`). One another solution is to create application and register that command on `I run a command "sylius:install:setup"` step, which is cleaner and makes executing commands more isolated.